pumpkin-world/src/biome.rs

```diff
@@ -2,13 +2,74 @@ use enum_dispatch::enum_dispatch;
 use serde::{Deserialize, Serialize};
 
 // TODO make this work with the protocol
+// TODO add missing biomes
 // Send by the registry
-#[derive(Serialize, Deserialize, Clone, Copy)]
+#[derive(Serialize, Deserialize, Clone, Copy, Default, Debug)]
 #[non_exhaustive]
 pub enum Biome {
```

Comment:
This Biome enum is more of an tmp thing. I wanted to parse biome from a JSON.

Reply:
Ok. Its needs remove or i can nothing to do?